google-cloud-firestore/Rakefile

```diff
@@ -21,6 +21,7 @@ require "rake/testtask"
 desc "Run tests."
 Rake::TestTask.new do |t|
   t.libs << "test"
+  t.options = "-v"
```

Comment:
Do we still need this flag anymore? I seem to remember this was added while debugging the flaky unit test on Windows.

Reply:
Removed the verbose statements from this PR.